sktime/clustering/_time_series_k_means.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+"""Time Series K-Means Clusterer."""
+
+__author__ = ["Christopher Holder", "Tony Bagnall"]
+__all__ = ["TimeSeriesKMeans"]
+
+from typing import List
+from sktime.clustering.base.base_types import (
```

Comment:
We need to make sure these are consistent with the existing types in `sktime/transformations/base.py` and the ones suggested in PR #964